src/components/editor/MonacoEditor/index.tsx

```diff
@@ -104,15 +95,36 @@ function MonacoEditor({
                     catalogType,
                 });
                 setStatus(EditorStatus.SAVING);
-                onChange(parsedVal, catalogName, catalogType)
-                    .then(() => {
-                        console.log('editor:update:saving:success');
-                        setStatus(EditorStatus.SAVED);
-                    })
-                    .catch(() => {
-                        console.log('editor:update:saving:failed');
-                        setStatus(EditorStatus.SAVE_FAILED);
+
+                if (editorSchemaScope) {
```

Comment:
This `if` is not totally needed but until we see stuff working well in production I want to keep it so the logging here can make it clear if something is scoped in the editor or not.